idea/idea-frontend-fir/src/org/jetbrains/kotlin/idea/frontend/api/fir/diagnostics/KtFirDataClassConverters.kt

```diff
@@ -995,6 +1015,12 @@ internal val KT_DIAGNOSTIC_CONVERTER = KtDiagnosticConverterBuilder.buildConvert
             token,
         )
     }
+    add(FirErrors.CONST_VAL_NOT_TOP_LEVEL_OR_OBJECT) { firDiagnostic ->
```

Comment:
🤦  Ah, hope it will be added by someone else soon!

Reply:
Added here: https://github.com/JetBrains/kotlin/commit/1d42c7005696df6188c28d689c28df9b4dcb5604